pkg/controller/kogitoruntime/deployer.go

```diff
@@ -143,6 +141,18 @@ func onDeploymentCreate(cli *client.Client, deployment *v1.Deployment, kogitoSer
 	if kogitoService.GetStatus().GetExternalURI() != "" {
 		framework.SetEnvVar(envVarExternalURL, kogitoService.GetStatus().GetExternalURI(), &deployment.Spec.Template.Spec.Containers[0])
 	}
+	// create sa
+	if err := createServiceAccountIfNotExists(cli, kogitoService.GetNamespace()); err != nil {
+		return fmt.Errorf("error creating Service Account: %v", err)
+	}
+	// create role
+	if err := createRoleIfNotExists(cli, kogitoService.GetNamespace()); err != nil {
+		return fmt.Errorf("error creating role: %v", err)
+	}
+	//create rolebinding
+	if err := createRoleBindingIfNotExists(cli, kogitoService.GetNamespace()); err != nil {
+		return fmt.Errorf("error creating rolebinding: %v", err)
+	}
```

Comment:
Do you mind creating a function to handle this creation? or we can just move this logic to the `KogitoRuntime` controller instead, before using the API to deploy the service. This RBAC creation feature doesn't seems in the right place here.

Reply:
Moved to kogitoRuntime_controller